test/e2e/scheduling/taint_based_evictions.go

```diff
@@ -21,7 +21,7 @@ import (
 	"fmt"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Please revert this line. There is no need to create an alias here.

Reply:
Yeah, I revert these changes in this PR. But the `v1` prefix is added by `goimports` which related to https://github.com/kubernetes/kubernetes/issues/77045 and https://github.com/kubernetes/kubernetes/pull/77407 :D

Reply:
Yeah it's goimport does this automatically.